src/wallet/wallet.cpp

```diff
@@ -2792,215 +2790,167 @@ bool CWallet::CreateTransactionInternal(
             CFeeRate discard_rate = GetDiscardRate(*this);
 
             // Get the fee rate to use effective values in coin selection
-            CFeeRate nFeeRateNeeded = GetMinimumFeeRate(*this, coin_control, &feeCalc);
+            coin_selection_params.effective_fee = GetMinimumFeeRate(*this, coin_control, &feeCalc);
```

Comment:
`effective_feerate`, since it's a feerate not a fee.

Reply:
Existing name, won't change for now.